<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QLabel" name="lbUserName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbPassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
